### ISSUE TITLE
Import more short descriptions

### ISSUE THREAD

**Issue by ddbeck** [mdn/short-descriptions#14]
Now that we've got scripts to support this work, I need to revise and open pull requests to add more short descriptions (for properties known to mdn/data) to this repo.

**Comment by ddbeck** [comment]
Following a discussion with @wbamberg, I'm going to attempt to scrape standard, non-experimental properties into the repo this sprint. I'll be opening PRs with 15 short descriptions each. We'll aim to release as many as we can when we get to #21, also during this sprint.

**Comment by ddbeck** [comment]
This work is continuing in this sprint, aiming to merge 60 or more CSS properties during the sprint. At the start of the sprint, this is where we're at:

  | Count | Progress
-- | -- | --
**All properties** | 424 | 100.00%
Revised | 227 | 53.54%
Linted | 188 | 44.34%
Done | 182 | 42.92%
*Merged* | 88 | 20.75%
  |   |  
**Standard** | 321 | 100.00%
Revised | 226 | 70.40%
Linted | 183 | 57.01%
Done | 181 | 56.39%
*Merged* | 88 | 27.41%
  |   |  
**Standard and stable** | 238 | 100.00%
Revised | 226 | 94.96%
Linted | 157 | 65.97%
Done | 156 | 65.55%
*Merged* | 81 | 34.03%

**Comment by jmswisher** [comment]
I'm assuming this is on hold for the moment. Let me know if that's not correct.

**Comment by atopal** [comment]
Yes, it's on hold. Daniel, can you give an update on where we are right now?

**Comment by ddbeck** [comment]
@atopal The stats are unchanged from above. We have 88 short descriptions in the repo (and in the package on npm). There are some more additions unreviewed in #29. The discussions last week pointed to this being lower priority than BCD's CSS goals and the stumptown experiments.

**Comment by atopal** [comment]
Thanks for the update, Daniel. The prioritization is correct, but we need to decide what the future for this project is @wbamberg
